src/errno.rs

```diff
@@ -1910,6 +1925,172 @@ mod consts {
     }
 }
 
+#[cfg(target_os = "haiku")]
```

Comment:
Nix does not define constants anymore.  Instead, all of these should be defined in the libc crate, and nix should reference libc.

Reply:
You mean how the [linux version](https://github.com/nix-rust/nix/blob/master/src/errno.rs#L529) is done? I had wondered why it was different to the other platforms.

Reply:
Yeah.  It looks like Somebody fixed the Linux definitions in change 3354ffe91dcc217c6ac62f88d5e54a1ea34e5f7a , but not other OS's.